frontend/src/lib/lemon-ui/icons/icons3000.stories.tsx

```diff
@@ -0,0 +1,92 @@
+import * as packageIcons from '@posthog/icons'
+import { Meta, StoryObj } from '@storybook/react'
+import { copyToClipboard } from 'lib/utils/copyToClipboard'
+
+import { LemonCollapse } from '../LemonCollapse'
+import { Tooltip } from '../Tooltip'
+import { ELEMENTS, OBJECTS, TEAMS_AND_COMPANIES, TECHNOLOGY } from './categories'
+
+const meta: Meta = {
+    title: 'PostHog 3000/Icons',
+    tags: ['test-skip'],
+    parameters: {
+        previewTabs: {
+            'storybook/docs/panel': {
+                hidden: true,
+            },
+        },
+    },
+}
+export default meta
+
+const posthogIcons = Object.entries(packageIcons)
+    .filter(([key]) => key !== 'BaseIcon')
+    .map(([key, Icon]) => ({ name: key, icon: Icon }))
+
+const IconTemplate = ({ icons }: { icons: { name: string; icon: any }[] }): JSX.Element => {
+    const onClick = (name: string): void => {
+        void copyToClipboard(name)
+    }
+
+    return (
+        <div className="grid grid-cols-6 gap-4">
+            {icons.map(({ name, icon: Icon }) => {
+                return (
+                    <div onClick={() => onClick(name)} key={name} className="flex justify-center">
+                        <Tooltip title="Click to copy">
+                            <div className="flex flex-col items-center space-y-2 max-w-24 py-2 px-4 cursor-pointer rounded hover:bg-secondary-3000">
+                                <Icon className="w-10 h-10" />
+                                <span className="text-xs">{name}</span>
+                            </div>
+                        </Tooltip>
+                    </div>
+                )
+            })}
+        </div>
+    )
+}
+
+export function Alphabetical(): JSX.Element {
+    return <IconTemplate icons={posthogIcons} />
+}
+
+const GroupBase = ({ group }: { group: Record<string, string[]> }): JSX.Element => {
+    return (
+        <LemonCollapse
+            multiple
+            panels={Object.entries(group).map(([key, icons]) => {
```

Comment:
Would be even more handy if the panels were expanded by default (using `defaultActiveKeys`), what do you think?

Reply:
I considered that originally but some of the categories were quite large and knocked the others off the page. Happy to follow up with it because I'm still undecided as to which is better